### PR TITLE
Fix sending TOKEN to cozy-controller in tests

### PR DESCRIPTION
**Pull request by nono**
It fixes tests for cozy-monitor!

### CODE REVIEW

File: server/lib/paas.coffee [hunk 1]
```diff
@@ -46,7 +46,8 @@ class AppManager
 
     # Get token from token file if in production mode.
     getAuthController: ->
-        if process.env.NODE_ENV is 'production'
+        if process.env.NODE_ENV is 'production' or
```
Comment (kosssi) [inline server/lib/paas.coffee]: if process.env.NODE_ENV in ['production', 'test']

Comment (nono) [inline server/lib/paas.coffee]: Yes, it's a bit easier to read. But I prefered to be coherent with the rest of cozy-controller. For example, we have the same check here: https://github.com/cozy/cozy-home/blob/master/server/models/application.coffee#L36

Comment (kosssi) [inline server/lib/paas.coffee]: If I can do better I do it ;) according as you want.

Comment (frankrousseau) [inline server/lib/paas.coffee]: It's matter of taste bug I prefer the @kosssi version.

**Comment by frankrousseau** [comment]
The style change broke the tests.

**Pull request merged**